### ISSUE TITLE
CloudShell blocked on "Select directory..." message

### ISSUE THREAD

**Issue by yevster** [microsoft/vscode-azure-account#659]
I tried to use the CloudShell integration for the first timed. I signed in through the browser, and then saw the "Select directory..." message with nothing under it. Nothing I do in the shell seems to bypass that message. This happens for both Bash and PowerShell.

VSCode Version: 1.39.1
OS: Windows 10 x64, build 18362

**Comment by alexweininger** [comment]
Closing as stale. If this is still reproducing please reopen this issue, or open a new issue.
